ecosystem/platform/server/app/models/user.rb

```diff
@@ -70,7 +72,7 @@ def self.create_new_user_from_oauth(auth)
   def maybe_send_ait2_registration_complete_email
     return unless ait2_registration_complete?
 
-    SendRegistrationCompleteEmailJob.perform_now({ user_id: id })
+    SendRegistrationCompleteEmailJob.perform_later({ user_id: id })
```

Comment:
I am in favor of this change but I believe the code was written such that it needs to be sent immediately so that any exceptions could propagate to the view (e.g. email doesn't exist). Can you double check that?